lib/Cake/Utility/Hash.php

```diff
@@ -384,8 +384,8 @@ public static function remove(array $data, $path) {
  * following the path specified in `$groupPath`.
  *
  * @param array $data Array from where to extract keys and values
- * @param array|string $keyPath A dot-separated string or array for formatting rules.
- * @param array|string $valuePath A dot-separated string or array for formatting rules.
+ * @param array|string|array $keyPath A dot-separated string or array for formatting rules.
```

Comment:
This starts with `array|`. What is this fixing?

Reply:
This fixes errors reported by phpstan.

Reply:
Sorry, wrong branch.